reversion/models.py

```diff
@@ -299,6 +312,14 @@ def revert(self):
     def __str__(self):
         return self.object_repr
 
+    def __repr__(self):
+        return 'Version(pk={!r}, revision={!r}, object_id={!r}, content_type={!r})'.format(
+            self.pk,
+            getattr(self, 'revision', None),
+            self.object_id,
+            getattr(self, 'content_type', None),
```

Comment:
Why are we using getattr?

Reply:
Because otherwise you might get an `RelatedObjectDoesNotExist` error if the ForeignKey is None.